recipe/meta.yaml

```diff
@@ -14,13 +14,14 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<37]
```

Comment:
To be clear python 3.6 is still supported by the maintainers https://github.com/flask-admin/flask-admin/blob/3f9bda85b3a6040ee0327474fbf5e3515467b58a/setup.py#L51. Even if we do not build for `py<37` we can skip `py<36`